lib/components/split-pane.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable quote-props */
```

Comment:
[divider_horizontal](https://github.com/zeit/hyper/blob/master/lib/components/split-pane.js#L159) and family aren't camel case identifiers. Preferably I would've changed it to be `divider-horizontal` instead, but it would be a bit rash as someone might be using the underscore separated identifiers in a plugin or similar.

The reason it's not `dividerHorizontal` is this: https://github.com/zeit/hyper/blob/master/lib/components/split-pane.js#L86 (could of course capitalize though)